src/Projection/PdoEventStoreProjector.php

```diff
@@ -633,8 +638,15 @@ private function createProjection(): void
 EOT;
 
         $statement = $this->connection->prepare($sql);
-        // we ignore any occuring error here (duplicate projection)
-        $statement->execute([$this->name, $this->status->getValue()]);
+        try {
+            $statement->execute([$this->name, $this->status->getValue()]);
+        } catch (\PDOException $exception) {
```

Comment:
As you noticed already, this lib expects currently ERRMODE_SILENT, which is the default anyway. So I think we should remove the catching of exception here and only check for errorCode on the statement after query.